src/make.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function make() {
+    let r = Array.from(arguments);
+  
```

Comment:
Try to name variables with clear names
For this case it's better to name this variable `let arr =  Array.from(arguments);`